vm-bootstorm/ReadMe.md

```diff
@@ -0,0 +1,65 @@
+##################################################################################################
+### VM BOOTSTORM ###
+##################################################################################################
+<b>DESCRIPTION</b>
+This template deploys requested number of VMs, plus a controller VM with public IP address, and a Virtual Network. Controller VM then shut-down all VMs then boot them simultaneously to measure an average VM boot time.
+
+For controller VM to manage all VMs, Azure SPN needs to be configured using instructions given below.
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fdipakmsft%2Fazure-quickstart-templates%2Fmaster%2Fvm-bootstorm%2Fazuredeploy.json" target="_blank">
```

Comment:
Does this work?
<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fvm-bootstorm%2Fazuredeploy.json" target="_blank">
    <img src="http://azuredeploy.net/deploybutton.png"/>
</a>